### PR TITLE
Create vi_vn

### PR DESCRIPTION
**Pull request by godkyo98**

### CODE REVIEW

File: src/main/resources/assets/thermoo/lang/en_us.json [hunk 1]
```diff
@@ -4,7 +4,6 @@
     "attribute.thermoo.generic.frost_resistance": "Frost Resistance",
     "attribute.thermoo.generic.heat_resistance": "Heat Resistance",
     
-
     "commands.thermoo.temperature.exception.not_living_entity": "Target is not a living entity!",
```
Comment (TheDeathlyCow) [inline src/main/resources/assets/thermoo/lang/en_us.json:7]: why was this line removed?

**Review by TheDeathlyCow — CHANGES_REQUESTED** [review]
Thanks for the translations, they look good except for one random line being removed from the English file?

File: src/main/resources/assets/thermoo/lang/en_us.json (hunk 1 above)
Comment (godkyo98) [reply src/main/resources/assets/thermoo/lang/en_us.json:7]: Actually, it will cause confusion when you update the new version later and be mistakenly compared with other translations

Comment (godkyo98) [reply src/main/resources/assets/thermoo/lang/en_us.json:7]: Right now I'm finishing up with scorchful, you can do the same. This is very confusing

Comment (TheDeathlyCow) [reply src/main/resources/assets/thermoo/lang/en_us.json:7]: What are you confused about?

Comment (godkyo98) [reply src/main/resources/assets/thermoo/lang/en_us.json:7]: Well, it's easy to understand because I compare the lines to see if you have updated the new version or made any changes. With a small number like 32-33 lines, it's simple. But with 300 words poured on it, it's very difficult

Comment (godkyo98) [reply src/main/resources/assets/thermoo/lang/en_us.json:7]: Because it's in the pull section 🤣

**Review by TheDeathlyCow — APPROVED** [review]
